astropy/time/formats.py

```diff
@@ -680,6 +681,43 @@ class TimeUnix(TimeFromEpoch):
     epoch_format = 'iso'
 
 
+class TimeUnixTai(TimeUnix):
+    """
+    Seconds from 1970-01-01 00:00:08 TAI (see notes), including leap seconds.
+
+    This will generally differ from Unix time by the cumulative integral number
+    of leap seconds since 1970-01-01 UTC.  This convention matches the definition
+    for linux CLOCK_TAI (https://www.cl.cam.ac.uk/~mgk25/posix-clocks.html).
+
+    Caveats:
+
+    - Before 1972, fractional adjustments to UTC were made, so the difference
+      between ``unix`` and ``unix_tai`` time is no longer an integer.
+    - Because of the fractional adjustments, to be very precise, ``unix_tai``
+      is the number of seconds since ``1970-01-01 00:00:08 TAI`` or equivalently
+      ``1969-12-31 23:59:59.999918 UTC``.  The difference between TAI and UTC
+      at that epoch was 8.000082 sec.
+    - On the day of a leap second the difference between `unix` and `unix_tai`
```

Comment:
This should fix the warning.
```suggestion
    - On the day of a leap second the difference between ``unix`` and ``unix_tai``
```